supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -79,6 +78,18 @@
 
 *See also*:
 
+[discrete]
+[[raw-interpreted-program]]
+==== image:images/yes.png[yes] raw-interpreted program (noun)
+
+*Description*: _Raw-interpreted language programs_ are programs that do not need to be compiled and are directly executed by a language interpreter.
```

Comment:
Should "language" be removed from this, since the term doesn't have it?

```suggestion
*Description*: _Raw-interpreted programs_ are programs that do not need to be compiled and are directly executed by a language interpreter.
```

Reply:
+1